tests/testmongodb.nim

```diff
@@ -0,0 +1,219 @@
+import unittest
+
+import os, strutils, sequtils, times
+
+import norm / mongodb
+
+
+const
+  # for local testing, modify your /etc/hosts file to contain "mongodb_1"
+  # pointing to your local mongodb server
+  dbConnection = "mongodb://mongodb_1:27017"
+  dbName = "TestDb"
+  customDbName = "TestCustomDb"
+
+type
+  Copyright = object
+    legal_name: string

```

Comment:
Pleas stick to camelCase for codebase consistency.

Reply:
making the change